network/stations.csv

```diff
@@ -1720,6 +1720,7 @@ RIVW,XX,Riverview College,-33.8278,151.1584,9999,,WGS84,1996-01-31T00:00:00Z,199
 RIZ,NZ,Raoul Island,-29.2449,-177.9289,110,,WGS84,2009-05-18T00:00:00Z,9999-01-01T00:00:00Z
 RKAC,SC,Rakaia School,-43.751461192,172.02314733,107,,WGS84,2005-02-23T00:00:00Z,9999-01-01T00:00:00Z
 RKIA,XX,Rakaia,-43.300327621,171.310110376,473,,WGS84,1995-11-09T00:00:00Z,1996-04-19T00:00:00Z
+RLGLK,EN,Raoul Green Lake,-29.26068,-177.918038,30,,WGS84,2023-01-30T02:59:59Z,9999-01-01T00:00:00Z
```

Comment:
since this site is co-located with `GLKZ`, I suggest to use the same `Name`
can you please change `Raoul Green Lake` to `Green Lake` ?

GLKZ entry for this is:
`GLKZ,KI,Green Lake,-29.26068,-177.918038,30,,WGS84,2009-05-02T00:00:00Z,9999-01-01T00:00:00Z`